src/controllers/dashboard/scheduledtasks/scheduledtasks.js

```diff
@@ -90,16 +90,23 @@ define(["jQuery", "loading", "events", "globalize", "serverNotifications", "huma
         return html;
     }
 
+    function setTaskButtonIcon(button, icon) {
+        var inner = button.querySelector("i");
+        inner.classList.remove("stop");
+        inner.classList.remove("play_arrow");
```

Comment:
```suggestion
        inner.classList.remove("stop", "play_arrow");
```

This should work as well and is slightly cleaner (I'm not sure how it acts when one of the classes isn't found though, so better test it)